packages/history/index.ts

```diff
@@ -0,0 +1,437 @@
+////////////////////////////////////////////////////////////////////////////////
+//#region TYPES
```

Comment:
I've never seen the `#region` thing before. Is that specific to some editor plugin? or recognized natively in vscode?

Reply:
yeah - built into vscode: https://code.visualstudio.com/docs/editor/codebasics#_folding